content/brazilian-portuguese/android-tictactoe/puzzle1.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Quebra-cabeça 1 - Execute o jogo e resolva-o"
+date: 2021-03-13T14:17:07.42-07:00
+draft: false
+weight: 2
+---
+
+## Execute o jogo
+Pressione o botão play (►) na parte superior do Android Studio para iniciar o jogo.
+<img src="../resources/_gen/images/run_app.png" height="30%" width="30%" title="Botão Executar aplicativo" alt="Botão Executar aplicativo"/>
+
+Você verá que o título na parte superior do aplicativo diz Toe-Tac-Tic (isso não parece certo!); vamos mudar para Tic-Tac-Toe.
```

Comment:
@ozhang22 Tic-Tac-Toe translated to Pt-BR is "Jogo da Velha". In this line, I suppose the working application downloaded by the reader already has this mistake so it is supposed to be fixed. Should we proceed by suggesting the student to change "Toe-Tic-Tac" to "Jogo da Velha", to "Tic-Tac-Toe" or we should do something else?

Reply:
N/A

Reply:
I think this is OK, let's have the student change Toe-Tic-Tac to Jogo da Velha